### PR TITLE
Fix AppImageLauncher package

### PR DESCRIPTION
**Pull request by korewaChino**
This PR adds `libappimageupdate` and fixes AppImageLauncher, adding back postinstall and postuninstall scripts and fixes the libappimageupdate dependency.

### CODE REVIEW

File: anda/lib/libappimageupdate/libappimageupdate.spec [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+%{_prefix}/lib/cmake/AppImageUpdate/
+
+
+
+
+
+
+%changelog
+* Tue Oct 25 2022 Cappy Ishihara <cappy@cappuchino.xyz>
+-
```
Comment (windowsboy111) [inline anda/lib/libappimageupdate/libappimageupdate.spec:80]: ?

Comment (korewaChino) [reply anda/lib/libappimageupdate/libappimageupdate.spec:75]: fixed

**Comment by korewaChino** [comment]
build still broken
dont merge yet

**Comment by korewaChino** [comment]
ok, patch now ready to merge

**Review by lleyton — DISMISSED** [review]

**Review by lleyton — APPROVED** [review]

**Review by lleyton — APPROVED** [review]

**Pull request merged**